specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2018-02-14/keyvault.json

```diff
@@ -902,6 +903,9 @@
         "tenantId",
         "sku"
       ],
+      "additionalProperties": {
+        "type": "string"
+      },
```

Comment:
I think this needs to go under the properties of the model definition.  Also do you have some example usage of this property?  As specified AdditionalProperties is a single string. Is this the intent?